### PR TITLE
Documentation for AVL Automation Tool

### PR DESCRIPTION
**Pull request by frede791**
This tool can be used to automatically generate the parameters required for the advanced lift drag plugin in Gazebo. The tool itself is already part of PX4 main and also includes this readme but I felt it would also be good to have this as part of documentation. 

This readme explains how to use the avl automation tool in detail. 

Link to code: https://github.com/PX4/PX4-Autopilot/tree/main/Tools/simulation/gz/tools/avl_automation

### CODE REVIEW

**Comment by hamishwillee** [comment]
FYI @frede791 I know this is in draft but I have done a minimal subedit for things I know you would have to change. 
1. All markdown docs are only allowed one level deep - i.e. inside `en/subfolder/doc_here.md`.
   That makes it much easier to link to particular docs because the links to a doc are always relative and always follow the same pattern.
2. I use a single sentence per line, and format with prettier. The sentence thing makes it much easier to add github comments because you're commenting a sentence rather than a paragraph.
3. You can't write a tag format like `<this is my example>.pdf>` without wrapping it in backticks - otherwise it is an xml or html tag, and will not render properly.
4. I moved the file and added to the summary so it appears in sidebar. Might not be right place to put it.

File: en/sim_gazebo_gz/tools_avl_automation.md [hunk 1]
```diff
@@ -0,0 +1,141 @@
+# Advanced Lift Drag (AVL) Automation Tool
```
Comment (hamishwillee) [inline en/sim_gazebo_gz/tools_avl_automation.md:1]: Note, dropped to level 1 heading and expanded AVL.

File: en/sim_gazebo_gz/tools_avl_automation.md [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+
+This tool automates the writing of the Advanced Lift Drag plugin used by the Advanced Plane by automatizing the coefficient generation and requiring minimal user calculations.
```
Comment (hamishwillee) [inline en/sim_gazebo_gz/tools_avl_automation.md:3]: It isn't clear WHY you need to do this.

File: en/sim_gazebo_gz/tools_avl_automation.md [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+
+## Setup
+
+In order to run this tool, it is necessary to follow these steps:
+
+1. Download AVL 3.36 from <https://web.mit.edu/drela/Public/web/avl/>.
+   The file for AVL version 3.36 can be found about halfway down the page.
+2. After downloading, extract AVL and move it to the home directory using:
+
+```sh
```
Comment (hamishwillee) [inline en/sim_gazebo_gz/tools_avl_automation.md:13]: FYI only, I am starting to use `sh` to indicate "generic shell" through the docs. `shell` is probably even more correct, but I'd rather stick with `sh` and change it globally than have it mixed across the docs.

File: en/sim_gazebo_gz/tools_avl_automation.md [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+sudo tar -xf avl3.36.tgz
+mv ./Avl /home/
+```
+
+Follow the README.md found in Avl to finish the setup process for AVL (requires to set up plotlib and eispack libraries).
+I recommend using the gfortran compile option.
```
Comment (hamishwillee) [inline en/sim_gazebo_gz/tools_avl_automation.md:19]: Use third person
```suggestion
We recommend using the gfortran compile option.
```

File: en/sim_gazebo_gz/tools_avl_automation.md [hunk 5]
```diff
@@ -0,0 +1,141 @@
...
+| Cnq         | Cenq                              | dCn/dq (yaw moment slope wrt pitching rate)              |
+| CXr         | CDr                               | dCD/dr (drag coefficient slope wrt yaw rate)             |
+| CYr         | CYr                               | dCY/dr (side force slope wrt yaw rate)                   |
+| CZr         | CLr                               | dCL/dr (lift coefficient slope wrt yaw rate)             |
+| Clr         | Cellr                             | dCl/dr (roll moment slope wrt yaw rate)                  |
+| Cmr         | Cemr                              | dCm/dr (pitching moment slope wrt yaw rate)              |
+| Cnr         | Cenr                              | dCn/dr (yaw moment slope wrt yaw rate)                   |
+
+Furthermore, every control surface also has six own parameters, which are also derived from this log file.
+`{i}`` below ranges from 1 to the number of unique control surface types in the model.
```
Comment (hamishwillee) [inline en/sim_gazebo_gz/tools_avl_automation.md:132]:
```suggestion
`{i}` below ranges from 1 to the number of unique control surface types in the model.
```

**Comment by frede791** [comment]
@hamishwillee I had noticed the folder error but I hadn't yet found the time for fixing it, which is why I had left it as draft. Thanks for fixing it!

File: en/sim_gazebo_gz/vehicles.md [hunk 6]
```diff
@@ -60,8 +60,11 @@ make px4_sitl gz_advanced_plane
 ![Advanced Plane in Gazebo](../../assets/simulation/gazebo/vehicles/advanced_plane.png)
 
 :::note
-The difference between the Advanced Plane and the "regular plane" lies in the Lift Physics that the two models use.
-For more detail on the lift calculations for the Advanced Plane, see: https://github.com/PX4/PX4-SITL_gazebo-classic/blob/20ded0757b4f2cb362833538716caf1e938b162a/src/liftdrag_plugin/README.md
+The difference between the Advanced Plane and the "regular plane" lies in the Lift Physics that the two models use:
+
+- You can configure the _Advanced Lift Drag_ plugin used by the model to more closely match a particular vehicle using the [Advanced Lift Drag Tool](../sim_gazebo_gz/tools_avl_automation.md).
```
Comment (hamishwillee) [inline en/sim_gazebo_gz/vehicles.md:65]: @frede791 This line is new

File: en/sim_gazebo_gz/tools_avl_automation.md [hunk 7]
```diff
@@ -0,0 +1,162 @@
+# Advanced Lift Drag (AVL) Automation Tool
+
+The Gazebo [Advanced Plane](../sim_gazebo_gz/vehicles.md#advanced-plane) vehicle model uses the _Advanced Lift Drag_ plugin to model vehicle lift and drag behaviour.
+This tool allows you to calculate the parameters required to create a _Advanced Lift Drag_ plugin for your own particular vehicle.
```
Comment (hamishwillee) [inline en/sim_gazebo_gz/tools_avl_automation.md:4]: FYI This is a more direct explanation of purpose - you use this to customize advanced plane for your purpose

File: en/sim_gazebo_gz/tools_avl_automation.md [hunk 8]
```diff
@@ -0,0 +1,162 @@
...
+1. Copy the example `input.yml` to `<your_custom_yaml_file>.yml` and modify it to match your desired plane
+1. Run the tool on your yml file:
+
+   ```sh
+   python input_avl.py <your_custom_yaml_file>.yml
+   ```
+
+   Note that the `yaml` and `argparse` packages must be present in your Python environment.
+
+1. The tool prompts for a range of vehicle specific parameters that are needed in order to specify the geometry and physical properties of the plane.
```
Comment (hamishwillee) [inline en/sim_gazebo_gz/tools_avl_automation.md:56]: I pulled this bit out of "Functionality" as it is part of "what you do as a user". The functionality bit is now purely "what the tool does". It is good to separate like this.

**Comment by hamishwillee** [comment]
@frede791 I have subedited mostly to make it clear when you use the tool. I think this is good but can you give it one more quick scan to check my changes didn't modify your intent.

**Comment by frede791** [comment]
LGTM!

**Pull request merged**